.travis/matrix-script.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+set -ev
+
+source .travis/utils.sh
+
+run_tests() {
+    TESTS=${TEST_OVERRIDE:-"$1"}
+
+    ENV_VARS=""
+    if [ $# -eq 2 ]; then
+        ENV_VARS="$2"
+    fi
+
+    if [ -z ${COMMAND_OVERRIDE} ]; then
+        travis_runner web_test ".travis/test_runner.sh $TESTS"
+    else
+        travis_runner web_test $COMMAND_OVERRIDE
```

Comment:
The `override` vars here allow changing what gets run if you're running it locally - see `simulate.sh`